src/docker-utils.coffee

```diff
@@ -145,7 +145,7 @@ do ->
 							return true
 						return containerHasExited(containerInfo.Id)
 				.map (containerInfo) ->
-					docker.getContainer(containerInfo.Id).removeAsync()
+					docker.getContainer(containerInfo.Id).removeAsync(v: true, force: true)
```

Comment:
That check will only happen for supervisor containers (see the check for supervisor tags right above)